pkg/compute/init.go

```diff
@@ -118,6 +117,10 @@ func (c *InitCommand) Exec(in io.Reader, out io.Writer) (err error) {
 			StarterKits: c.Globals.File.StarterKits.AssemblyScript,
 			Toolchain:   NewAssemblyScript(),
 		}),
+		NewLanguage(&LanguageOptions{
+			Name:        "other",
+			DisplayName: "Other (pre-compiled WASM binary)",
```

Comment:
```suggestion
			DisplayName: "Other ('bring your own' build process)",
```

Reply:
@fgsch I see you've disagreed with @triblondon's suggestion. Do you have an alternative for consideration (or can you elaborate on why the suggestion isn't ideal)?

Reply:
I like the original "pre-compiled WASM binary" text.